README.md

```diff
@@ -1,66 +1,128 @@
-If you see a `400 Bad Request - HTTP` error it may be necessary to add http:// URLs to your device origins and return fields. To do this, go to your amazon developer AVS home page (https://developer.amazon.com/avs/home.html) and from `My products` select `manage` and then `security profile`. Now add http://localhost:3000 to the `Allowed origins` and http://localhost:3000/authresponse to the `Allowed return URLs`. Now refresh the browser window with the original error.
+   for the xCORE VocalFusion XVF3100.
 
-9. Enter `sudo reboot` to reboot the Raspberry Pi and complete the installation.
+   Read and accept the AVS Device SDK license agreement.
+
+8. You will be asked whether you want the Sample App to run automatically when the Raspberry Pi boots. It is recommended that you respond "yes" to this option.
+
+9. Read and accept the Sensory license agreement. Wait for the script to complete the installation. The script is configuring the Raspberry Pi audio system, downloading and updating dependencies, building and configuring the AVS Device SDK. It takes around 30 minutes to complete.
+
+10. Enter `sudo reboot` to reboot the Raspberry Pi and complete the installation.
+
+11. If you selected the option to run the Sample App on boot you should now be able to complete the registration by following the steps from 2 onward here:
+https://github.com/alexa/avs-device-sdk/wiki/Raspberry-Pi-Quick-Start-Guide-with-Script#finish-authorization-using-login-with-amazon
+
+12. Now you can execute an AVS command such as "Alexa, what time is it?".
+
+   On the XMOS **xCORE VocalFusion XVF3510 Kit for Amazon AVS**, the LED on the Pi HAT board will change colour when the system hears the "Alexa" keyword, and will then cycle back and forth whilst waiting for a response from the Amazon AVS server.
+
+   On the XMOS **xCORE VocalFusion Stereo 4-Mic Kit for Amazon AVS**, the LEDs on the development board should reflect the approximate direction from which the microphones are receiving a stimulus.
 
 ## Running the AVS SDK Sample App
-The following commands will be available to run from a terminal:
+The automated installation script creates a number of aliases which can be used to execute the AVS Device SDK client, or run the unit tests:
 - `avsrun` to run the Sample App.
-- `avsauth` to re-authenticate your Alexa device details.
-- `avsunit` to run the unit tests.
-- `avssetup` to re-install the Sample App.
+- `avsunit` to run the unit tests (invoke Amazon's `test.sh`).
+- `avssetup` to re-install the Sample App (re-run XMOS modified `setup.sh`).
+
+## Using different Amazon details
+To change client and product ID, download your updated config.json and in the same folder run `avssetup config.json rebuild`. Subsequent `avsrun` calls will use the new details.
```

Comment:
I would re-test this as well.

Reply:
Yes, this doesn't work for the same reason as above.